addons/stock/models/product.py

```diff
@@ -94,7 +94,7 @@ class Product(models.Model):
     @api.depends('stock_move_ids.product_qty', 'stock_move_ids.state')
     @api.depends_context(
         'lot_id', 'owner_id', 'package_id', 'from_date', 'to_date',
-        'company_owned', 'force_company',
+        'company_owned', 'company',
```

Comment:
Shouldn't we add 'allowed_company_ids' also @sle-odoo ?

Reply:
this field has a lot of side effects (i just listed some [here](https://github.com/odoo/odoo/pull/38273#issuecomment-540052954))

i think it is better it is only computed with the records of the main company

Reply:
Even in multi-company :thinking: ?
(Do we want to see all quantities on product form in multi-company or not :thinking: ?)

Reply:
it has side effects so i would say no at the moment

Reply:
Testing in https://github.com/odoo/odoo/pull/38304